src/main/java/uk/nhs/hee/tis/trainee/sync/service/TcsSyncService.java

```diff
@@ -70,13 +72,16 @@
 
   private final RestTemplate restTemplate;
 
+  private final AmazonSQS amazonSQS;
+
   private final Map<String, Function<Record, TraineeDetailsDto>> tableNameToMappingFunction;
 
   @Value("${service.trainee.url}")
   private String serviceUrl;
 
-  TcsSyncService(RestTemplate restTemplate, TraineeDetailsMapper mapper) {
+  TcsSyncService(RestTemplate restTemplate, TraineeDetailsMapper mapper, AmazonSQS amazonSQS) {
     this.restTemplate = restTemplate;
+    this.amazonSQS = amazonSQS;
```

Comment:
Looks like we can revert this class now the behaviour has been moved to the individual type services.